### PR TITLE
Re IDC #3077: Seg series thumbnail selection

### PR DESCRIPTION
**Pull request by GitanjaliChhetri**
### PR Checklist

- [ ] Brief description of changes
- [ ] Links to any relevant issues
- [ ] Required status checks are passing
- [ ] User cases if changes impact the user's experience
- [ ] `@mention` a maintainer to request a review

Fixes: https://github.com/OHIF/Viewers/issues/3077

### CODE REVIEW

**Comment by GitanjaliChhetri** [comment]
https://user-images.githubusercontent.com/112136540/212032949-9eddd4cc-55a4-4068-ad76-42389dd0e85f.mp4

https://user-images.githubusercontent.com/112136540/212033005-c892948f-7d5b-47e2-b819-f134b1726b92.mp4

**Comment by GitanjaliChhetri** [comment]
Fix for the other two bugs:
Show segmentation tolerance changed notification message only once after setting to a new value. 
Update thumbnail inconsistency warning for seg series after modifying segmentation tolerance value.

https://user-images.githubusercontent.com/112136540/212036877-cedf2cc7-513f-469e-8b1b-a92cd639a98c.mp4

File: extensions/dicom-segmentation/src/components/SegmentationSettings/SegmentationSettings.js [hunk 1]
```diff
@@ -141,29 +141,36 @@ const SegmentationSettings = ({ configuration, onBack, onChange, servicesManager
...
+                  return string.match(rgx);
+                };
 
-              if (validate(event.key)) {
-                event.preventDefault();
+                if (validate(event.key)) {
+                  event.preventDefault();
+                }
+
+                const {
```
Comment (igoroctaviano) [inline extensions/dicom-segmentation/src/components/SegmentationSettings/SegmentationSettings.js:154]: The message will be displayed only on enter or keypress? I thought this message should be triggered inside onChange. If the problem is multiple calls then you could set up a once function and execute it in the onChange e.g.
```
const once = fn => (...args) => {
  if (!fn) return;
  fn(...args);
  fn = null;
};
```

**Review by igoroctaviano — APPROVED** [review]
LGTM just left one comment but it's your call. Feel free to merge it if there's nothing there to be changed.

File: extensions/dicom-segmentation/src/components/SegmentationSettings/SegmentationSettings.js (hunk 1 above)
Comment (GitanjaliChhetri) [reply extensions/dicom-segmentation/src/components/SegmentationSettings/SegmentationSettings.js]: Thanks for the review, Igor!
I have implemented the once function for onChange as described.

**Pull request merged**